src/AdminCli/CliCommands/DropPriceCommand.cs

```diff
@@ -0,0 +1,52 @@
+using System;
+using AdminCli.HttpAccess;
+using AdminCli.NumberParsing;
+using McMaster.Extensions.CommandLineUtils;
+
+namespace AdminCli.CliCommands;
+
+public sealed class DropPriceCommand : ICliCommand
+{
+    public DropPriceCommand(IHttpService httpService) => HttpService = httpService;
+
+    private IHttpService HttpService { get; }
+
+    public void ConfigureCommand(CommandLineApplication config)
+    {
+        config.Description = "Drops the price of a specified product.";
+        var productIdOption =
+            config.Option("-p|--productId <ProductId>",
+                          "The ID of the product whose price should be dropped. Must be a GUID.",
+                          CommandOptionType.SingleValue)
+                  .IsRequired();
+        var dropPriceByOption =
+            config.Option("-d|--dropPriceBy <Price>",
```

Comment:
Same as above, do not introduce different cases when building CLIs this makes them hard to remember